Applications/Spire/Include/Spire/Spire/Dimensions.hpp

```diff
@@ -52,6 +52,35 @@ namespace Spire {
     \return The translated point.
   */
   QPoint translate(const QPoint& p) noexcept;
+
+  //! Scales the pixel width at the system's current DPI to 96 DPI.
```

Comment:
Fix the documentation.

Reply:
Do you mean to replace the word scales to the word unscales?

Reply:
More than just that, the documentation is incorrect, the function doesn't do what the documentation says it does.